Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,7 +10,56 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A."; 
+            }
+
+            var HighestBalanceEver = decimal.MinValue; 
+            var PersonWithHighestBalance = new StringBuilder() ; 
+
+            for (int i = 0; i <= peopleAndBalances.Length - 1; i++)
+			{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4929752)